tests/integration/features/traffic_limiting.feature

```diff
@@ -0,0 +1,46 @@
+  Feature: Backup & Restore sources scenario with traffic limit. 
+
+  Background:
+    Given default configuration
+    And a working s3
+    And a working zookeeper on zookeeper01
+    And a working clickhouse on clickhouse01
+    And clickhouse on clickhouse01 has test schema
+    And clickhouse01 has test clickhouse data test1
+    And we have executed queries on clickhouse01
+    """
+    CREATE USER test_user IDENTIFIED WITH plaintext_password BY 'password';
```

Comment:
Let's test only data table without ACl entites. In my opinion they are complicating test. SImplify as far as possible